lib/matplotlib/rcsetup.py

```diff
@@ -1310,6 +1310,7 @@ def _validate_linestyle(ls):
     'xtick.minor.bottom':    [True, validate_bool],    # draw x axis bottom minor ticks
     'xtick.major.top':   [True, validate_bool],  # draw x axis top major ticks
     'xtick.major.bottom':    [True, validate_bool],    # draw x axis bottom major ticks
+    'xtick.minor.default':  [4,validate_int],   #set default value for x axis minor ticks
```

Comment:
```suggestion
    'xtick.minor.default':  [4, validate_int],   #set default value for x axis minor ticks
```

Please adhere to the pep8 style conventions. We enforce these to keep the codebase clean and readable. If you check the CI runs at the bottom of the PR https://github.com/matplotlib/matplotlib/pull/14355, you'll notice that the flake8 run in travis complains on some codestyle issues.